internal/configs/sharding/allowlist.go

```diff
@@ -0,0 +1,47 @@
+package shardingconfig
+
+import (
+	"fmt"
+
+	bls_cosi "github.com/harmony-one/harmony/crypto/bls"
+)
+
+type Allowlist struct {
+	MaxLimitPerShard int
+	BLSPublicKeys    []bls_cosi.PublicKeyWrapper
+}
+
+func _BLS(pubkeys []string) []bls_cosi.PublicKeyWrapper {
```

Comment:
Pardon my ignorance but why does this start with an underscore?

Reply:
removed